src/agent/onefuzz-agent/src/tasks/fuzz/libfuzzer_fuzz.rs

```diff
@@ -35,7 +39,7 @@ const RESYNC_PERIOD: Duration = Duration::from_secs(30);
 const PROC_INFO_COLLECTION_DELAY: Duration = Duration::from_secs(1);
 
 // Period of reporting proc info about running processes.
-const PROC_INFO_PERIOD: Duration = Duration::from_secs(30);
+const PROC_INFO_PERIOD: Duration = Duration::from_secs(10);
```

Comment:
Is this change related?

Reply:
Debugging artifact.  How we collect processor stats for libfuzzer targets is timer based, which was frustrating during debugging of this issue.  I've reset this, but we should address how processor stats are collected in the future.